torch_xla/experimental/pallas_kernels/quantized_matmul_kernel.py

```diff
@@ -388,6 +388,38 @@ def quantized_matmul_int8(
     (6, 32, 8192, 3584, 'bfloat16', True): (128, 1024, 3584),
     (6, 64, 1280, 8192, 'bfloat16', True): (128, 256, 8192),
     (6, 64, 8192, 1024, 'bfloat16', True): (128, 2048, 1024),
+    (6, 16, 3584, 3584, 'bfloat16', True): (128, 896, 3584),
```

Comment:
there are 32 rows. The ticket mentioned 11 batch sizes: [256, 512, 1024 ... 16384](https://screenshot.googleplex.com/ufAacBKT77pmTVj). So there should be 44 rows. I guess you need to tune for more batch sizes?

Reply:
Done. Added more. Now there are 44 rows.